packages/client/hmi-client/src/components/widgets/tera-infinite-canvas.vue

```diff
@@ -63,6 +66,10 @@ function handleZoom(e: any, container: d3.Selection<SVGGElement, any, null, any>
 		.style('transform', `translate(${e.transform.x}px, ${e.transform.y}px) scale(${e.transform.k})`)
 		.style('transform-origin', '0 0');
 
+	dashArrayYOffset.value = e.transform.y * -1;
+	dashArrayGapSize.value = d3.scaleLog().domain([0.1, 10]).range([10, 40])(e.transform.k);
+	dashArrayStrokeWidth.value = d3.scaleLog().domain([0.1, 10]).range([2, 6])(e.transform.k);
```

Comment:
I get some slowdown when I am panning/zooming while these lines are enabled. I think we should add this dotted pattern in a similar way to how the grid was rendered (worth investigating a more efficient method).